### ISSUE TITLE
`warning: output filename collision` on installation

### ISSUE THREAD

**Issue by emilbonnek** [rvben/rumdl#61]
Whenever I cargo install or update rumdl I get this error:

```
warning: output filename collision.
The bin target `rumdl` in package `rumdl v0.0.125` has the same output filename as the lib target `rumdl` in package `rumdl v0.0.125`.
Colliding filename is: C:\Users\emilb\AppData\Local\Temp\cargo-install3DFCuW\release\rumdl.pdb
The targets should have unique names.
Consider changing their names to be unique or compiling them separately.
This may become a hard error in the future; see <https://github.com/rust-lang/cargo/issues/6313>.
```

it is because the lib, and the bin are both called rumdl.
I have seen other packages name their lib with a '_lib' suffix.
Could you consider doing something similar to avoid this warning? 🙏

**Comment by rvben** [comment]
I'll look into that! Thanks for raising the issue, @emilbonnek.

**Comment by rvben** [comment]
Thanks again @emilbonnek, I've taken your suggestion and added the '_lib' suffix so we don't encounter the issue anymore.

Let me know when you hit anything else, or see opportunities for improvements.
